srcpkgs/gargoyle/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'gargoyle'
+pkgname=gargoyle
+version=2023.1
+revision=1
+build_style=cmake
+configure_args="-DWITH_TTS=$(vopt_if tts ON OFF) -DWITH_QT6=false"
+hostmakedepends="cmake fmt pkg-config"
+makedepends="SDL2-devel SDL2_mixer-devel qt5-devel fontconfig-devel
+	freetype-devel libjpeg-turbo-devel libpng-devel zlib-devel
+	$(vopt_if tts speech-dispatcher-devel)"
+depends="SDL2 SDL2_mixer $(vopt_if tts speech-dispatcher)"
```

Comment:
SDL2 and SDL2_mixer probably don't need to be listed here. If the programming links to the shared library the dependency should be detected